mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Scrivener.Mixfile do
 
   defp deps do
     [
-      {:ecto, "~> 1.0"},
+      {:ecto, ">= 1.0.0"},
```

Comment:
Oh, nice. Instead of use `or` that will force to use `~> 2.0.beta`, I updated to use `>= 1.0.0 and < 3.0.0` that will automatically get the final release.